docs/UserGuide.md

```diff
@@ -1,37 +1,66 @@
 
-# Vendor Vault User Guide
+# VendorVault User Guide
 
-Vendor Vault is a **desktop app for managing supplier contact information and deliveries, optimized for use via a  Line Interface** (CLI). If you can type fast, VendorVault can get your contact management tasks done faster than traditional GUI apps. Vendor Vault is targeted at small convenience/grocery stores. 
+VendorVault is a **desktop app for managing supplier contact information and deliveries, optimized for use via a Command Line Interface** (CLI). If you can type fast, VendorVault can get your contact management tasks done faster than traditional GUI apps. Vendor Vault is targeted at small convenience/grocery stores. 
```

Comment:
Should there be a spacing between Vendor Vault on the last line/

Reply:
Could the last line be more descriptive? Such as 'VendorVault specifically targets small convenience/grocery stores and aims to help them achieve higher efficiency!'